README.md

```diff
@@ -0,0 +1,125 @@
+# Understanding a Temporal workflow as an implementation of the Actor Model
+
+The project demonstrates how key features of the Actor Model map onto Temporal workflows.
+
+|NOTE|
+|----|
+|In order to get full benefit from using this project, it's useful to have a basic understanding of what Temporal is and how it works. [This tutorial](https://docs.temporal.io/temporal) provides a good basic introduction to the Temporal framework.|
```

Comment:
This isn't exactly a tutorial, how about this page?

Reply:
Which page? Not see a link? @angelazhou32

Reply:
https://docs.temporal.io/temporal You marked it as 'this tutorial' in this sentence:

[This tutorial](https://docs.temporal.io/temporal) provides a good basic introduction to the Temporal framework.|

Reply:
Changed to "overview", addressed